templates/rhpam70-kieserver-externaldb.yaml

```diff
@@ -83,116 +124,111 @@ parameters:
+  name: EXECUTION_SERVER_HOSTNAME_HTTPS
   value: ''
   required: false
-- displayName: Maven repository URL
-  description: Fully qualified URL to a Maven repository.
-  name: MAVEN_REPO_URL
-  value: ''
+- displayName: KIE Server Keystore Secret Name
+  description: The name of the secret containing the keystore file
+  name: KIE_SERVER_HTTPS_SECRET
+  value: kieserver-app-secret
   required: false
-- displayName: Maven repository username
-  description: Username to access the Maven repository.
-  name: MAVEN_REPO_USERNAME
-  value: ''
+- displayName: KIE Server Keystore Filename
+  description: The name of the keystore file within the secret
+  name: KIE_SERVER_HTTPS_KEYSTORE
+  value: keystore.jks
   required: false
-- displayName: Maven repository password
-  description: Password to access the Maven repository.
-  name: MAVEN_REPO_PASSWORD
-  value: ''
+- displayName: KIE Server Certificate Name
+  description: The name associated with the server certificate
+  name: KIE_SERVER_HTTPS_NAME
+  value: jboss
   required: false
-- displayName: "Datasource Minimum Pool Size"
-  description: "Sets xa-pool/min-pool-size for the configured datasource."
-  name: DB_MIN_POOL_SIZE
+- displayName: KIE Server Keystore Password
+  description: The password for the keystore and certificate
+  name: KIE_SERVER_HTTPS_PASSWORD
+  value: mykeystorepass
   required: false
-- displayName: "Datasource Maximum Pool Size"
-  description: "Sets xa-pool/max-pool-size for the configured datasource."
-  name: DB_MAX_POOL_SIZE
-  required: false
-- displayName: "Datasource Transaction Isolation"
-  description: "Sets transaction-isolation for the configured datasource."
-  name: DB_TX_ISOLATION
-  required: false
-- displayName: KIE Server Persistence Dialect
-  description: KIE execution server persistence dialect (Sets the org.kie.server.persistence.dialect system property)
-  name: KIE_SERVER_PERSISTENCE_DIALECT
-  value: org.hibernate.dialect.PostgreSQLDialect
-  required: false
-- displayName: KIE Server Persistence DS
-  description: KIE execution server persistence datasource (Sets the org.kie.server.persistence.ds system property)
-  name: KIE_SERVER_PERSISTENCE_DS
-  value: java:/jboss/datasources/dbe
+- displayName: KIE Server Bypass Auth User
+  description: KIE execution server bypass auth user (Sets the org.kie.server.bypass.auth.user system property)
+  name: KIE_SERVER_BYPASS_AUTH_USER
+  value: 'false'
   required: false
-- displayName: KIE Server Persistence TM
-  description: KIE execution server persistence transaction manager (Sets the org.kie.server.persistence.tm system property)
-  name: KIE_SERVER_PERSISTENCE_TM
-  value: org.hibernate.engine.transaction.jta.platform.internal.JBossAppServerJtaPlatform
+- displayName: "Timer service data store refresh interval (in milliseconds)"
+  description: "Sets refresh-interval for the EJB timer service database-data-store."
```

Comment:
wouldn't be better change to EJB timer database-data-store service?

Reply:
@spolti Filippe, not sure I follow... can you elaborate on this, please?

Reply:
update the description from Sets refresh-interval for the EJB timer service database-data-store. to Sets refresh-interval for the EJB timer database-data-store service.